### PR TITLE
Add JD150 and JD200

### PR DESCRIPTION
**Pull request by arash-binary**
https://redmine.deriv.cloud/issues/31128#JumpDiffusion_150_and_200_(JD150/JD200)_for_EU_Market

### CODE REVIEW

**Review by junbon-binary — APPROVED** [review]

File: share/underlyings.yml [hunk 1]
```diff
@@ -546,6 +531,51 @@ JD75:
   quoted_currency: ~
   submarket: jump_index
   symbol: JD75
+JD100:
+  asset: JD100
+  display_name: Volatility 100 Jump diffusion
```
Comment (junbon-binary) [inline share/underlyings.yml:536]: The `display_name` format has been to `Jump ## Index`. For more information: https://redmine.deriv.cloud/issues/33801#jump_diffusion_display_name

File: share/underlyings.yml [hunk 2]
```diff
@@ -546,6 +531,51 @@ JD75:
...
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  market_type: non_financial
+  pip_size: 0.01
+  quoted_currency: ~
+  submarket: jump_index
+  symbol: JD100
+JD150:
+  asset: JD150
+  display_name: Volatility 150 Jump diffusion
```
Comment (junbon-binary) [inline share/underlyings.yml:551]: please refer to https://github.com/binary-com/perl-Finance-Underlying/pull/81/files#r644497885

File: share/underlyings.yml [hunk 3]
```diff
@@ -546,6 +531,51 @@ JD75:
...
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  market_type: non_financial
+  pip_size: 0.01
+  quoted_currency: ~
+  submarket: jump_index
+  symbol: JD150
+JD200:
+  asset: JD200
+  display_name: Volatility 200 Jump diffusion
```
Comment (junbon-binary) [inline share/underlyings.yml:566]: please refer to https://github.com/binary-com/perl-Finance-Underlying/pull/81/files#r644497885

File: share/underlyings.yml (hunk 1 above)
Comment (arash-binary) [reply share/underlyings.yml]: I've updated the repos. Thanks for the notice 🙏

**Pull request merged**